sunkit_pyvista/plotter.py

```diff
@@ -97,6 +124,7 @@ def plot_map(self, m, **kwargs):
         cmap = kwargs.pop('cmap', m.cmap)
         mesh = self._pyvista_mesh(m)
         self.plotter.add_mesh(mesh, cmap=cmap, **kwargs)
+        self._plotter.camera_position = 'xz'
```

Comment:
What's this for?

Reply:
This doesn't do anything, I needed some input on how we position the camera first so I left this in for now

Reply:
I'm still a bit confused on how we set the inital plot, whether we leave it in it's rotated position or forcefully set `roll` to a particular value to make the plot upright?

Reply:
I think we should set `roll` to the correct value to make sure the solar north pole is pointing up.